stacks/ecs.yml

```diff
@@ -588,6 +590,64 @@ Resources:
         - Key: Project
           Value: Infrastructure
       Timeout: 300
+  ASGSizerRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      Policies:
+        - PolicyName: "ASGSizerLambdaPolicy"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: "Allow"
+                Resource: "*"
+                Action:
+                  - logs:CreateLogGroup
```

Comment:
These log actions are dupes with the `AWSLambdaBasicExecutionRole` being included